docs/docs/topics/schema.mdx

```diff
@@ -14,7 +15,61 @@ Out of the box, Infrahub doesn't have a schema for most things and it's up to us
 
 Unlike traditional databases that can only have one schema at a time, in Infrahub it is possible to have a different schema per branch. This is possible because the schema itself is stored in the database like any other object.
 
-There are several ways to [load a new schema](/guides/import-schema).
+## Schema file
+
+The recommended way to manage and load a schema is to create a schema file in YAML format. With a schema file it's possible to:
```

Comment:
It's not necessarily the recommended way. It's just a way of doing it together with the repository method.

Arguably, in a production environment, you might want to have the schema in an external repository and let Infrahub import the changes from it. 

`infrahubctl schema load` is great when you are developing a schema or trying to test something, or when you want to make one of changes, which might be required during some migrations.

Reply:
@wvandeun @BaptisteGi - Currently, under the Guides section, we have a 'Creating a Schema' page that covers a lot, including how to import the schema file. Then we have the next page, 'Schema File', which also covers how to import a schema file. That's why I thought it would be good to move the contents of that page under the 'Schema' topics. I didn't change anything except for adding 'How to generate the token'.